### DISCUSSION TITLE
Change of teacher to student developer pack

### DISCUSSION THREAD

**Discussion by Dhanasri101** [community/community#110985] [Packages]
### Select Topic Area

General

### Body

Accidently i have choosen github teacher developer pack instead of student developer pack...can you please let me know to to change it from teacher to student?

**Comment by SebastienXia** [comment]
Hello, @Dhanasri101 

Personally, I didn't find anything for your problem there, I suggest you open a support ticket at Github: 
🔗 https://support.github.com/features/education
